services/app-api/one-seed.json

```diff
@@ -4720,17 +4720,17 @@
         "url": "https://uploads-develop-attachments-116229642442.s3.us-east-1.amazonaws.com/protected/us-east-1%3A86a190fe-b195-42bf-9685-9761bf0ff14b/1655844289419/Attachment%203.1-A%2C%20%234b%2C%20Page%203f%20Track.pdf"
       }
     ],
-    "componentId": "MD.5533.R00.00",
+    "componentId": "MD-5533.R00.00",
```

Comment:
I only changed this single base waiver format as it is the only approved status waiver whereby you can add a temp extension. The parent waiver needed to be same format as required for TE in order for things to function 100% properly. I assume other base waiver formats will be updated when we get to the base waiver for package view id update story.

Reply:
That sounds to me like the right approach... for package view, you can assume a conversion from the submission data that can/will include ID updates